package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/cli",
-  "version": "1.0.0",
+  "version": "1.0.0-rc.0",
```

Comment:
Should this be rc.0?

Reply:
What do you mean? It is. You mean without a dash? There should be a dash.

Reply:
I meant should this be 1.0.0-rc.6 or 1.0.0-rc.7 instead of 1.0.0-rc.0

Reply:
No, this is a different package. They will never all align.